source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -62,6 +62,11 @@ class ZipkinSpan : public Tracing::Span {
    */
   void setTag(const std::string& name, const std::string& value) override;
 
+  /**
+   * Do nothing for now. Log records could potentially be translated to tags in the future.
+   */
+  void log(SystemTime /*timestamp*/, const std::string& /*event*/) override {}
```

Comment:
In zipkin, this would be represented by an annotation, e.g. https://github.com/envoyproxy/envoy/blob/master/source/extensions/tracers/zipkin/tracer.cc#L22 but the annotation value would be the `event`. Would be good if you could add this impl as part of the PR.

Reply:
+1